store/src/db.rs

```diff
@@ -159,12 +163,17 @@ impl ChainDB {
             verified: Some(true),
             txs_fees: vec![],
         };
+        let block_tx_stat = BlockTxStat {
+            txs_size: vec![],
+            txs_cycles: vec![],
+        };
 
         attach_block_cell(&db_txn, &genesis)?;
         let last_block_hash_in_previous_epoch = epoch.last_block_hash_in_previous_epoch();
 
         db_txn.insert_block(genesis)?;
         db_txn.insert_block_ext(&genesis_hash, &ext)?;
+        db_txn.insert_block_tx_stat(&genesis_hash, &block_tx_stat)?;
```

Comment:
This only inserts for genesis, we have to insert the tx stats for other blocks as well.

Reply:
It‘s not necessary to insert the tx stats for genesis.

Reply:
i will remove the line